index.html

```diff
@@ -212,5 +212,35 @@ <h2>About me</h2>
                 </div>
             </div>
         </section>
+        <section class="contact-form">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form action="https://formspree.io/account" method="post">
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/rabayarabu/Portfolio-setup-and-mobile-first/pull/4/files/3ac15008e74e4d7c52c3163cb6549838b6258c91#r1153119752), please make sure to collect form data using Formspree service as mentioned in the project requirements 🎯 You can follow [this YouTube tutorial](https://www.youtube.com/watch?v=sVPlb5C3oQ8) to learn more about form submission using Formspree service ✨ Right now, I am getting the following error 👇

![image](https://user-images.githubusercontent.com/106119105/228835576-5487a76e-ed0a-452d-94b5-7dda9350d359.png)